src/plugins/dependencies-exact-version/cmd.js

```diff
@@ -10,10 +11,14 @@ const semver = require('semver')
  * @param {String} key : defines if we are focusing on dependencies or devDependencies
  */
 const checkVersion = (badVersions, key) => {
-    for (const dependency in global.packagefile[key]) {
-        const depVersion = (global.packagefile[key][dependency]).replace('workspace:', '')
-        if (!semver.valid(depVersion)) {
-            badVersions[key][dependency] = depVersion
+    // only check version in package.json
+    // if no package-lock.json was found
+    if (!fs.existsSync(path.join(process.env.QUALSCAN_PROJECT_PATH, 'package-lock.json'))) {
```

Comment:
Would also be good to check for `yarn.lock` file.

Reply:
Is it better now?